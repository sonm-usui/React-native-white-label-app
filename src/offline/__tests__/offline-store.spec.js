// @flow
import 'react-native'

import offlineReducer, { offline } from '../offline-store'

describe('Offline-store', () => {

  it('action: Offline, value: False', () => {
    expect(offlineReducer(undefined, offline(false))).toMatchSnapshot()
  })

  it('action: Offline, value: True', () => {
    expect(offlineReducer(undefined, offline(true))).toMatchSnapshot()
  })
})
