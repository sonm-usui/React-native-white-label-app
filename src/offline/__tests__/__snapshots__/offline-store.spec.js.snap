// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Offline-store action: Offline, value: False 1`] = `
Object {
  "offline": false,
}
`;

exports[`Offline-store action: Offline, value: True 1`] = `
Object {
  "offline": true,
}
`;
